privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -73,59 +61,39 @@ def train_shadow_classifiers(
         :param num_elements_per_classes: specific class distribution
         :return: list of shadow classifiers,
                  accuracies for the classifiers
-        :rtype: Tuple[  List[:class:`.art.estimators.estimator.BaseEstimator`],
-                        List[float]]
+        :rtype: List[:class:`.art.estimators.estimator.BaseEstimator`]
         """
-
         shadow_classifiers = []
-        accuracies =[]
-
         num_classes = len(num_elements_per_classes)
-
         for shadow_training_set in shadow_training_sets:
-            shadow_training_X, shadow_training_y = shadow_training_set
-            train_X, test_X, train_y, test_y = train_test_split(
-                shadow_training_X, shadow_training_y, test_size=0.3
-            )
-            train_set = (train_X, train_y)
-            test_set = (test_X, test_y)
-
-            model = FCNeuralNet()
-            accuracy = trainer(
-                train_set, test_set, num_elements_per_classes, model
+            model = FCNeuralNet(self.input_shape)  # TODO: infer model structure from the target model adaptively
```

Comment:
Again, no code with todo-statements will be merged into main!

Reply:
I deleted the TODO statement.